apps/x509.c

```diff
@@ -541,6 +557,19 @@ int x509_main(int argc, char **argv)
         print_name(bio_err, "subject=", X509_REQ_get_subject_name(req),
                    nmflag);
 
+        if (subj) {
+  	    X509_NAME *n = parse_name(subj, chtype, multirdn); 
+            if (!n) {
```

Comment:
blank line between the declaration and code.  also test against NULL and don't use ! for pointers.